README.md

```diff
@@ -84,6 +84,7 @@ magnification.dll |`PInvoke.Magnification`| [![NuGet](https://buildstats.info/nu
 mscoree.dll  |`PInvoke.MSCorEE` | [![NuGet](https://buildstats.info/nuget/PInvoke.MSCorEE)](https://www.nuget.org/packages/PInvoke.MSCorEE)|.NET Framework CLR host
 msi.dll      |`PInvoke.Msi`     | [![NuGet](https://buildstats.info/nuget/PInvoke.Msi)](https://www.nuget.org/packages/PInvoke.Msi)|[Microsoft Installer][Msi]
 fusion.dll   |`PInvoke.Fusion`  | [![NuGet](https://buildstats.info/nuget/PInvoke.Fusion)](https://www.nuget.org/packages/PInvoke.Fusion)|.NET Framework Fusion
+iphlpapi.dll |`PInvoke.IPHlpApi`| [![NuGet](https://buildstats.info/nuget/PInvoke.IPHlpApi)](https://www.nuget.org/packages/PInvoke.IPHlpApi)|[IP Helper](IPHlpApi)
```

Comment:
The sort order is wrong, probably due to `fusion` being missorted as well, as you've already fixed in another PR. That'll cause a merge conflict too. :(

Reply:
Ah, silly me 😄 . I'll rebase & fix this when the other PRs which are already approved get merged.